pkg/governance-scripts/contracts/BalancerContractRegistryInitializer.sol

```diff
@@ -70,14 +73,21 @@ contract BalancerContractRegistryInitializer {
         _authorizer = IBasicAuthorizer(address(vault.getAuthorizer()));
     }
 
+    /**
+     * @notice The function that initializes the Balancer contract registry, based on the data supplied on deployment.
+     * @dev This function can only be called once. This contract must be granted permission to call two functions on
+     * the `BalancerContractRegistry` being initialized: `registerBalancerContract` and
+     * `addOrUpdateBalancerContractAlias`. If this is not done, it will revert with `PermissionNotGranted`.
+     *
+     * Note that this contract revokes these permissions when the initialization is complete, so this does not need
+     * to be done externally.
+     */
     function initializeBalancerContractRegistry() external {
         // Explicitly ensure this can only be called once.
         if (_initialized) {
             revert AlreadyInitialized();
         }
 
-        _initialized = true;
-
```

Comment:
this really doesn't change much.

Yes, it saves some gas if you can't perform one of the actions, but as a general rule it's not important to optimize for that case. It's better to keep it clear and straightforward.

In this case the change is also fine, no need to change it back.